script/deploy.s.sol

```diff
@@ -122,12 +122,18 @@ contract DeployerScript is Create2Helper, ArtifactsReader {
         // deploy KYCViewer
         (viewer, viewerImpl) = deployKYCViewer();
 
+        // Replace KintoID with the factory
```

Comment:
what do you mean by this?

Reply:
This Kinto ID has the factory address properly set